src/modules/navigator/CMakeLists.txt

```diff
@@ -34,6 +34,10 @@
 px4_add_module(
 	MODULE modules__navigator
 	MAIN navigator
+	COMPILE_FLAGS
+		-Wno-cast-align # TODO: fix and enable
```

Comment:
This is how it is Implemented in the  CMakeLists.txt off mavlink_receiver.cpp.
This will need fixfin in the future